els-back/database/seeders/AdminSeeder.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Admin;
+use Illuminate\Database\Seeder;
+
+class AdminSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Admin::create([
```

Comment:
Pls use upsert to avoid errors or repetitive data when running seeders over and over again. If you could also apply upsert of categories seeder will be great.